idl/github.com/uber/cadence/replicator.thrift

```diff
@@ -53,7 +53,7 @@ struct HistoryTaskAttributes {
   40: optional i64 (js.type = "Long") firstEventId
   50: optional i64 (js.type = "Long") nextEventId
   60: optional i64 (js.type = "Long") version
-  70: optional map<string, history.ReplicationInfo> replicationInfo
```

Comment:
Is this a backward compatible change?

Reply:
only the name is changed, as long as thrift's concrete implementation remains the same, we should be fine